includes/admin/class-wc-payments-admin.php

```diff
@@ -836,4 +837,25 @@ public function maybe_redirect_to_onboarding() {
 
 		$this->account->redirect_to_onboarding_page();
 	}
+
+	/**
+	 * Add woopay as a payment method to the edit order on admin.
+	 *
+	 * @param int $order_id order_id.
+	 */
+	public function show_woopay_payment_method_name_admin( $order_id ) {
+		$order = wc_get_order( $order_id );
+		if ( $order || ! $order->get_meta( 'is_woopay' ) ) {
```

Comment:
Should this `$order` be `! $order`?

Reply:
Fixed with 69cd2cce3e84a73031c82fd7094fe1dd38544312